packages/types/src/dtos/update-profile.dto.ts

```diff
@@ -0,0 +1,10 @@
+export interface UpdateProfileDTO {
```

Comment:
All attributes should be optional :
`firstName?: string;
  lastName?: string;
  username?: string;
  email?: string;
  bio?: string;
  password?: string;
  confirmPassword?: string;
  avatarURL?: string;`